### PR TITLE
LPS-166306 | LPS-166310

### PR DESCRIPTION
**Pull request by carolmariaabb**
:rotating_light: This PR contains some commits from https://github.com/liferay-objects/liferay-portal/pull/1266

Only the last **eight** commits are new. 

They are related to  [LPS-166306](https://issues.liferay.com/browse/LPS-166306) and [LPS-166310](https://issues.liferay.com/browse/LPS-166310)

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
:rotating_light: This PR contains some commits from https://github.com/liferay-objects/liferay-portal/pull/1266

Only the last **eight** commits are new. 

They are related to [LPS-166310](https://issues.liferay.com/browse/LPS-166310) and [LPS-166306](https://issues.liferay.com/browse/LPS-166306)

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi @carolmariaabb 
I finished my review. Just a minor change, let me know if you have questions :smile:

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java [hunk 1]
```diff
@@ -323,6 +329,46 @@ public ObjectEntry putByExternalReferenceCode(
...
+		throws Exception {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipService.getObjectRelationship(
+				_objectDefinition.getObjectDefinitionId(),
+				objectRelationshipName);
+
+		ObjectEntryManager objectEntryManager =
+			_objectEntryManagerTracker.getObjectEntryManager(
+				_objectDefinition.getStorageType());
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java]: Done!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by carolmariaabb** [comment]
This PR is waiting for https://github.com/brianchandotcom/liferay-portal/pull/125446 to be merged

**Pull request closed without merging**